### ISSUE TITLE
atom and atom-beta: undefined symbol udata_setCommonData_59

### ISSUE THREAD

**Issue by stites** [NixOS/nixpkgs#55789]
## Issue description
`atom` and `atom-beta` fail to run on the latest unstable nixos channel. This happens both when installing as root and as a user.

### Steps to reproduce
```
nix-channel --list   # confirming that nixpkgs is https://nixos.org/channels/nixpkgs-unstable
nix-channel --update
nix-env -e atom atom-beta # clearing out any old builds of atom
nix-env -i atom atom-beta
atom # or run atom-beta
```
Error for `atom` is
```
$ atom
/nix/store/yjbprgsll2l3xacpf1gcy324wdl1vll8-atom-1.33.0/share/atom/atom --executed-from=/home/stites --pid=22685: symbol lookup error: /nix/store/yjbprgsll2l3xacpf1gcy324wdl1vll8-atom-1.33.0/share/atom/atom --executed-from=/home/stites --pid=22685: undefined symbol: udata_setCommonData_59
```
and for `atom-beta`:
```
$ atom-beta 
/nix/store/2i57p21j0hs2316745k1ymczp77v4rl3-atom-beta-1.34.0-beta0/share/atom-beta/atom --executed-from=/home/stites --pid=22771: symbol lookup error: /nix/store/2i57p21j0hs2316745k1ymczp77v4rl3-atom-beta-1.34.0-beta0/share/atom-beta/atom --executed-from=/home/stites --pid=22771: undefined symbol: udata_setCommonData_59
```

## Technical details
```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.20.7, NixOS, 18.09.2137.168cbb39691 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(root): `"nixos-18.09.2137.168cbb39691, nixos-18.09-18.09.2137.168cbb39691, nixos-unstable-19.03pre169108.36f31600749, nixpkgs-18.09-18.09.1814.33000fe7cab"`
 - channels(stites): `"nixos-18.09-18.09.2137.168cbb39691, nixpkgs-19.03pre169106.1a88aa9e0cd, nixpkgs-unstable"`
 - nixpkgs: `/home/stites/.nix-defexpr/channels/nixpkgs`
```

**Comment by worldofpeace** [comment]
It appears that you're using the atom from `unstable` on `18.09`?
I think that's partially the problem.

I believe that this issue was a problem in an older version of electron and has now been fixed.

**Comment by stites** [comment]
@worldofpeace other services I rely on are borked on unstable (including X, it turns out). Could you point me in the direction of an issue so that I can fix this on 18.09? Searching for "atom" and "undefined" turns up nothing.

**Comment by stites** [comment]
A better plan is to invert my last thought and fix the borked things on 19.03-unstable. Confirming that Atom is working fine on unstable. Thank you @worldofpeace !

**Comment by worldofpeace** [comment]
Figured that it needed an update. You're welcome @stites :smile:
